### ISSUE TITLE
Duplicate index signature for type 'string'.

### ISSUE THREAD

**Issue by addiks** [Aksem/sprotty-routing-libavoid#3]
Hello,

I tried to install this module, but it seems that there is some conflict with sprotty.
Trying to transpile my typescript project (with tsc) results in this:

```
yarn run v1.22.19
$ tsc && webpack
node_modules/sprotty-routing-libavoid/node_modules/sprotty/lib/lib/jsx.d.ts:20:13 - error TS2374: Duplicate index signature for type 'string'.

20             [elemName: string]: VNodeData;
               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/sprotty/lib/lib/jsx.d.ts:20:13 - error TS2374: Duplicate index signature for type 'string'.

20             [elemName: string]: VNodeData;
               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

After every yarn change ( / change in node_modules) I have to manually open the file `node_modules/sprotty-routing-libavoid/node_modules/sprotty/lib/lib/jsx.d.ts` and comment the interface `IntrinsicElements` out:
```
declare global {
    namespace JSX {
        // interface IntrinsicElements {
        //     [elemName: string]: VNodeData;
        // }
    }
}
```
Only after manually commenting out these three lines does tsc work again.

Here is my `package.json`:

```
{
    "name": "[redacted]",
    "license": "MIT",
    "dependencies": {
        "@babel/core": "^7.0.0",
        "@babel/preset-env": "^7.11.5",
        "@types/bootstrap": "^5.2.10",
        "@types/node": "^12.11.7",
        "@types/uuid": "^9.0.8",
        "assert": "2.1.0",
        "babel-plugin-jsx-control-statements": "^4.1.0",
        "buffer": "^6.0.3",
        "circular-dependency-plugin": "^5.2.0",
        "colortranslator": "^1.1.1",
        "convert-units": "^3.0.0-beta.6",
        "css-loader": "^3.2.0",
        "inversify": "6.0.2",
        "node": "^12.12.0",
        "react": "^16.11.0",
        "react-dom": "^16.13.1",
        "reflect-metadata": "^0.1.13",
        "sha1": "^1.1.1",
        "source-map-loader": "^0.2.4",
        "sprotty": "1.2.0",
        "sprotty-elk": "1.2.0",
        "sprotty-protocol": "1.2.0",
        "sprotty-routing-libavoid": "^1.1.1",
        "style-loader": "^1.0.0",
        "ts-md5": "1.3.1",
        "typescript": "^5.4.5",
        "uuid": "^8.3.2",
        "webpack": "5.91.0"
    },
    "scripts": {
        "build": "php bin/console foo:typescript:generate-routing-json -v && tsc && webpack",
        "tsc": "tsc && webpack"
    },
    "devDependencies": {
        "ts-loader": "^6.2.1",
        "webpack-cli": "^3.3.9"
    }
}
```

My `yarn.lock`: https://nopaste.net/rSiIlMgCzJ

**Comment by Aksem** [comment]
Hi @addiks ,

thank you for reporting the issue. The problem is that sprotty-routing-libavoid  of pre 1.2 version was compatible only with sprotty 0.13. It's a known aspect that version of sprotty extension should match exactly version of sprotty used in the application. I just published version 1.2.1 compatible with sprotty 1.0.0 and as the next step, I'll publish another version compatible with sprotty ^v1.2.0 which you are using in your project. I'll let you know in this issue.

**Comment by Aksem** [comment]
Update: I just released sprotty-routing-libavoid 1.3.0, which is compatible with sprotty ^1.0, it means also with ^1.2. The next step will be an update of sprotty in the demo project to have a working example with sprotty 1.2.0. If you experience any further problems, feel free to reopen this issue or open a new one.
